File: ticket-system/src/app/core/model/Comment.ts
export class Comment {
  commentId?: number;
  ticketId: number;
  userId: number;
  replie: string;
  createAt?: string;
  fullName?: string;

  constructor() {
    this.ticketId = 0;
    this.userId = 0;
    this.replie = '';
  }
}

File: Ticket.Data/Models/Comment.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ticket.Data.Models
{
    public class Comment
    {
        public int CommentId { get; set; }
        public int UserId { get; set; }
        public int TicketId { get; set; }
        public string Replie { get; set; }
        public DateTime CreateAt { get; set; }
        public Ticket Ticket { get; set; } 
        public User User { get; set; }

    }
}
